src/I18n/Translate.elm

```diff
@@ -0,0 +1,24 @@
+module I18n.Translate exposing (Language(..), translate)
+
+import I18n.En exposing (enStrings)
+import I18n.Keys exposing (Key(..))
```

Comment:
Do you have elm-format installed in your editor? I would expect these imports to be alphabetised.